src/msw/radiobox.cpp

```diff
@@ -549,28 +537,24 @@ WX_FORWARD_STD_METHODS_TO_SUBWINDOWS(wxRadioBox, wxStaticBox, m_radioButtons)
 
 wxSize wxRadioBox::GetMaxButtonSize() const
 {
+    int cx1, cy1;
+    wxGetCharSize(m_hWnd, &cx1, &cy1, GetFont());
```

Comment:
This can be replaced with just `GetCharWidth()` call.

Reply:
There is a small difference in size, `GetCharWidth` is 1 pixel larger. Also after dividing, as `GetCharWidth` usually is even and `wxGetCharSize` odd. But it is still an improvement.

Reply:
Yes, we add one pixel when `wxDIALOG_UNIT_COMPATIBILITY` is off (which is the default), but I don't think it really matters here...